### DISCUSSION TITLE
How can I get balance of account in MyStrategy?

### DISCUSSION THREAD

**Discussion by foreverznb** [nautechsystems/nautilus_trader#1589] [Q&A] [answered]
Hi,

I'm very new to Nautilus Trader, sorry for the dumb question. 
I want to know the easiest way to get the account balance in a custom strategy. (self.context no longer working?)

**Comment by cjdsellers** [comment, marked as answer]
Hey @foreverznb

There's some docs on how to get an account for a certain venue [here](https://docs.nautilustrader.io/concepts/strategies.html#account-and-positional-information).

Once you have the account object there's various methods you can call, including for [balance](https://docs.nautilustrader.io/api_reference/model/objects.html#nautilus_trader.model.objects.AccountBalance) of a certain currency.

Hope that helps!

**Reply by foreverznb** [reply]
Thank you so much! Very helpful.
